library/Director/Objects/IcingaCommand.php

```diff
@@ -183,8 +183,13 @@ public function countDirectUses()
             array('n' => 'icinga_notification'),
             array('cnt' => 'COUNT(*)')
         )->where('n.command_id = ?', $id);
+        $qc = $db->select()->from(
+            array('c' => 'icinga_command_inheritance'),
+            array('cnt' => 'COUNT(*)')
```

Comment:
```suggestion
            [c' => 'icinga_command_inheritance'],
            ['cnt' => 'COUNT(*)']
```
We now use the short syntax of `array()` as it looks cleaner and modern